docs/user-guide/configuring/rules.md

```diff
@@ -160,6 +162,14 @@ You can also disable or enable specific rules for an entire file:
 alert('foo');
 ```
 
+To ensure that a rule is always applied (regardless of any future enable/disable lines):
+
+```js
+/* eslint no-alert: "off" */
+
+alert('foo');
+```
```

Comment:
Isn't it the opposite, this ensures that a rule is _not_ applied regardless of enable/disable comments?

Reply:
You're right, I'll go ahead and update that now